.dependabot/config.yml

```diff
@@ -0,0 +1,10 @@
+---
+version: 1
+update_configs:
+- package_manager: "ruby:bundler"
+  directory: "/"
+  update_schedule: "live"
+  version_requirement_updates: "off"
+  commit_message:
+    prefix: "No-Jira"
+    include_scope: true
```

Comment:
Missing newline here I think?

Reply:
looks like my script isn't adding the newline to the end 👎 i've gone ahead and added it